src/Features/Core/Portable/Common/TaggedTextStyle.cs

```diff
@@ -13,12 +11,14 @@ internal enum TaggedTextStyle
     {
         None = 0,
 
-        Strong = 0x1,
+        Strong = 1 << 0,
+
+        Emphasis = 1 << 1,
 
-        Emphasis = 0x2,
+        Underline = 1 << 2,
 
-        Underline = 0x4,
+        Code = 1 << 3,
 
-        Code = 0x8,
+        InlineCode = 1 << 4,
```

Comment:
I added a new TaggedTextStyle to differentiate `<c>` and `<code>`.

Currently `Code` value is `1000` and `InlineCode` is `10000`. Should `InlineCode` be `1001` or whatever value to share the same most significant bit? (i.e, indicate that InlineCode is a subset of Code)?

This will allow the following check: `(style & TaggedTextStyle.Code) == TaggedTextStyle.Code` to pass whether `style` is `Code` or `InlineCode`.

Reply:
The current approach seems fine to me.

Reply:
On second thought, it seems like the new addition should be:

```
PreserveWhitespace = 1 << 4,
```